flutter_application/lib/ServicePage/services_page.dart

```diff
@@ -28,45 +28,58 @@ connectService(serviceName) async {
   }
 }
 
-chooseConnectionServices(page) async {
+refreshPage(context) {
+  Navigator.pushNamed(
+      context,
+      '/home'
+  );
+}
+
+chooseConnectionServices(page, context) async {
   if (page == "Twitter" && buttonChoose.buttonChooseTwitter == false) {
     buttonChoose.buttonChooseTwitter = true;
     button.buttonConnectionTwitter = "Connected";
     buttoncheck.buttonConnectionTwitter = "Choose";
     buttonchoosecol.colbuttonChooseTwitter = colorConnected;
     buttoncol.colbuttonConnectionTwitter = const Color.fromARGB(255, 14, 41, 2);
+    refreshPage(context);
```

Comment:
why not put only one 'refreshPage' after all the if/else if ?

Reply:
yes you're right